docs/roadmap.md

```diff
@@ -46,6 +46,41 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 - [x] Interactive creation pipeline. [#1283](https://github.com/kubesphere/console/issues/1283)
 - [x] Add S2I webhook support. [#6](https://github.com/kubesphere/s2ioperator/issues/6)
 
+### KubeEdge Integration [#3070](https://github.com/kubesphere/kubesphere/issues/3070)
+
+- [ ] KubeEdge cloud components setup.
+- [ ] KubeEdge edge nodes setup.
+- [x] Edge nodes logging and metrics support.
+- [x] Automate edge node joining/leaving.
```

Comment:
"Automated edge node network configuration on joining/leaving" seems more clear?

Reply:
Automated network configuration on edge node joining/leaving ?

Reply:
Yes, this represents the function of edge-watcher with iptables-operator. Automate edge node joining is what we planned to do for more simplified joining process than keadm.

Reply:
I think we still missing one item which is `Scheduling workload to edge nodes`

Reply:
And one detail should also be fixed, automatic taint edge node on joining, not done yet. And workload of cloud, such as daemonsets should not be scheduled to edge nodes.

Reply:
@Ma-Dan pls review again and approve the pr with lgtm if no problem